### PR TITLE
【translate】mds/chunkserverclient

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/mds/chunkserverclient/chunkserver_client.h [hunk 1]
```diff
@@ -93,16 +95,16 @@ class ChunkServerClient {
         uint64_t sn);
 
     /**
-     * @brief 获取leader
+     * @brief get the leader
      * @detail
-     *   向目标chunkserver发送报文查询leader
+     *   send a message to the target chunk server to query the leader
```
Comment (bai-charisu) [inline src/mds/chunkserverclient/chunkserver_client.h:100]: chunkserver还是连这写吧

Comment (bai-charisu) [inline src/mds/chunkserverclient/chunkserver_client.h:100]: chunkserver还是连这写吧

File: src/mds/chunkserverclient/copyset_client.h [hunk 2]
```diff
@@ -60,42 +60,44 @@ class CopysetClient {
...
      *
-     * @param logicPoolId 逻辑池id
-     * @param copysetId 复制组id
-     * @param chunkId Chunk文件id
-     * @param correctedSn chunk不存在快照文件时需要修正的版本号
+     * @param logicPoolId
+     * @param copysetId
+     * @param chunkId
+     * @param correctedSn the version number that needs to be corrected when
+     *                    there is no snapshot file in the chunk
```
Comment (bai-charisu) [inline src/mds/chunkserverclient/copyset_client.h:71]: snapshot file for the chunk吧

File: src/mds/chunkserverclient/chunkserver_client.h (hunk 1 above)
Comment (SeanHai) [reply src/mds/chunkserverclient/chunkserver_client.h:100]: > chunkserver还是连这写吧

done

File: src/mds/chunkserverclient/copyset_client.h (hunk 2 above)
Comment (SeanHai) [reply src/mds/chunkserverclient/copyset_client.h:71]: > snapshot file for the chunk吧

done

**Review by bai-charisu — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**